README.md

```diff
@@ -25,7 +25,7 @@ SpinalHDL is:
  - Google group                   <br> https://groups.google.com/forum/#!forum/spinalhdl-hardware-description-language
  - Donation channel               <br> https://opencollective.com/spinalhdl
 
-[![Join the chat at https://gitter.im/SpinalHDL/SpinalHDL](https://badges.gitter.im/SpinalHDL/SpinalHDL.svg)](https://gitter.im/SpinalHDL/SpinalHDL?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) / 中文区[![Join the chat at https://gitter.im/SpinalHDL-CN/community](https://badges.gitter.im/SpinalHDL/SpinalHDL.svg)](https://gitter.im/SpinalHDL-CN/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+English: [![Join the chat at https://gitter.im/SpinalHDL/SpinalHDL](https://badges.gitter.im/SpinalHDL/SpinalHDL.svg)](https://gitter.im/SpinalHDL/SpinalHDL?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) 中文: [![Join the chat at https://gitter.im/SpinalHDL-CN/community](https://badges.gitter.im/SpinalHDL/SpinalHDL.svg)](https://gitter.im/SpinalHDL-CN/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
What do you think about putting them on two lines of the same paragraph?

```suggestion
English: [![Join the chat at https://gitter.im/SpinalHDL/SpinalHDL](https://badges.gitter.im/SpinalHDL/SpinalHDL.svg)](https://gitter.im/SpinalHDL/SpinalHDL?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)  
中文: [![Join the chat at https://gitter.im/SpinalHDL-CN/community](https://badges.gitter.im/SpinalHDL/SpinalHDL.svg)](https://gitter.im/SpinalHDL-CN/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Mind the 2 spaces at the end of the first line

Reply:
display them in one line are more compact way then two lines.
lines will increase the length of whole page.

Reply:
btw, what is the selectable prefix of the title? I see there are fix, docs, etc. Is there guidelines of them?

Reply:
It is from https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#-commit-message-guidelines, which is an extension of https://www.conventionalcommits.org/en/v1.0.0/

Reply:
It's understandable now.